modules/lx-autoscale/variables.tf

```diff
@@ -242,6 +242,12 @@ variable "CloudWatchAgentUrl" {
   default     = ""
 }
 
+variable "CloudWatchAppLogs" {
+  type        = "string"
```

Comment:
This ought to be a list I think. Then use `join()` to pass it as a comma-separated list to CFN.

Reply:
updated